### ISSUE TITLE
Selectable + transitions issue

### ISSUE THREAD

**Issue by cjaoude** [bootstrap-vue/bootstrap-vue#4370]
Possible bug...

Recently updated packages to the latest vue, bootstrap-vue. When updating the tables for certain breaking changes I encountered an issue with selectable rows + transitions. Using a `selectable` table/rows with transitions breaks select-ability. The select event doesn't appear to fire on the row.

After having the issue with my tables, I created a sanity-check table using the bootstrap-vue documentation table code for transitions. Then I added to it the code from the selectable example, and it breaks as well. Edit out `:tbody-transition-props` and select-ability works again. Gist example below:

Help resolving and/or confirmation that this is a bug appreciated. #

Sanity-check example table using documentation code:
https://gist.github.com/cjaoude/e85291d5b1413d849a61c5a87b86f15d

```
        "vue": "^2.6.10",
        "vue-template-compiler": "^2.6.10"
        "bootstrap": "^4.3.1",
        "bootstrap-vue": "^2.0.4",
```

**Comment by tmorehouse** [comment]
It appears that there is an issue with event delegation on `<tbody>` when it is rendered as a `<transition-group>` element (the row click handlers are delegated to the `<tbody>` instead of instantiated listeners on each row element)

**Comment by tmorehouse** [comment]
Will be fixed in release 2.1.0 (being released this week)

**Comment by tmorehouse** [comment]
BootstrapVue `v2.1.0` has been released
